assets/src/blocks/Covers/Covers.js

```diff
@@ -22,6 +22,41 @@ export class Covers extends Component {
       // Populate post types tokens for saved post types.
       let postTypeTokens = props.postTypesList.filter(post_type => props.post_types.includes(post_type.id)).map(post_type => post_type.name);
 
+      const { __ } = wp.i18n;
+
+      this.options = [{
+        label: __('Take Action Covers', 'p4ge'),
+        image: window.p4ge_vars.home + 'images/take_action_covers.png',
+        value: '1',
+        help: __('Take action covers pull the featured image, tags, have a 25 character excerpt and have a call to action button')
+      },
+        {
+          label: __('Campaign Covers', 'p4ge'),
+          image: window.p4ge_vars.home + 'images/campaign_covers.png',
+          value: '2',
+          help: __('Campaign covers pull the associated image and hashtag from the system tag definitions.'),
+        },
+        {
+          label: __('Content Covers', 'p4ge'),
+          image: window.p4ge_vars.home + 'images/content_covers.png',
+          value: '3',
+          help: __('Content covers pull the image from the post.')
+        }];
+
+      // Get current post type to filter LayoutSelector options in the case of "campaigns" post type.
+      let currentPostType = wp.data.select('core/editor').getCurrentPostType();
+
+      if ('campaign' === currentPostType)
```

Comment:
You can do the assignment inside of this if (and add an else) so that it doesn't get assigned and overwritten immediately for campaigns.

Reply:
Or maybe better: create the array with the common element first, then add the others if we're not on a campaign. That way you don't have to duplicate the definition of the common option.